azure-pipelines/code-coverage/msal-code-cov.yml

```diff
@@ -0,0 +1,67 @@
+# File: azure-pipelines\code-coverage\msal.yml
+# Description: Code coverage for msal
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+# Variable: 'msalBranchName' msal branch to be used when running the pipeline manually
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
```

Comment:
do we need both pipelines to run on PR?
`pr: none`

Reply:
Not at the moment - this will be done in the near future when we have a nightly run